test/e2e/index.js

```diff
@@ -65,68 +76,179 @@ describe("Application launch", function () { // eslint-disable-line func-names
-            await app.client.setValue("#username", config.username);
-            await app.client.setValue("#password", config.password);
-            await app.client.click("button=Sign in");
-            await app.client.waitForVisible(".notification-message");
-            const loginError = await app.client.getText(".notification-message");
-            assert.equal(loginError, "Incorrect username or password", "should show error for not exists user");
+            await app.client.setValue(loginScreen.loginUsername, config.username);
+            await app.client.setValue(loginScreen.loginPassword, config.password);
+            await app.client.click(loginScreen.signin);
+            await app.client.waitForVisible(loginScreen.notificationMessage);
+            const loginError = await app.client.getText(loginScreen.notificationMessage);
+            assert.equal(loginError, "User doesn't exist", "should show error for not exists user");
         });
     });
 
     describe("Registration", () => {
         let seed;
 
-        it("Step 1", async () => {
-            await app.client.click("button=Sign up");
-            await app.client.setValue("#username", config.username);
-            await app.client.setValue("#password", config.password);
-            await app.client.setValue("#conf_password", config.password);
-            await app.client.click("button=Next");
-            await app.client.waitUntil(async () => app.client.isExisting("#seed"), config.timeoutForElementChecks);
-            const seedExists = await app.client.isExisting("#seed");
+        it("TC:565: Username length validation", async () => {
+            await app.client.click(loginScreen.signup);
+            await app.client.setValue(
+                registrationScreen.registraionUsernameID,
+                config.veryLongUsernameOne + config.veryLongUsernameTwo,
+            );
+            const username = await app.client.getValue(registrationScreen.registraionUsernameID);
+            assert.equal(username, config.veryLongUsernameOne + config.veryLongUsernameOne);
+        });
+
+        it("TC:564: Signup with special characters for username", async () => {
+            await app.client.setValue(registrationScreen.registraionUsernameID, config.usernameWithSpace);
+            await app.client.setValue(registrationScreen.registrationPasswordId, config.password);
+            await app.client.setValue(registrationScreen.registrationConfirmPassword, config.password);
+            await app.client.click(registrationScreen.registrationNextButton);
+            await app.client.waitUntil(
+                async () => app.client.isExisting(registrationScreen.errorForm),
+                config.timeoutForElementChecks,
+            );
+            const error = await app.client.getText(registrationScreen.errorForm);
+            assert.equal(error, statusCodes.EXCEPTION_USERNAME_WRONG_FORMAT(0, 0));
+            console.log("Checked username with space");
```

Comment:
Better to use such structure here: remove console.log and use it() test wrapper arrond necessary assert to substitute it. Afterwards it("TC:564:..) will become describe("TC:564...) block